src/lammpsparser/output.py

```
@@ -105,7 +105,7 @@ def parse_lammps_output(
     Returns:
         dict: Nested dictionary with two top-level keys:
 
-        - ``"generic"`` – quantities stored in pyiron/ASE units:
+        - ``"generic"`` – quantities stored in ASE units:
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)